File: ejercicios-js/ejercicio3.js
const palabras = [
  "Aceituna", "Murciélago", "Educación", "Aeropuerto",
  "Otorrinolaringólogo", "Euforia", "Aceite",
  "Paleontólogo", "Arquitectura", "Hipopótamo"
];

const vocales = ['A', 'E', 'I', 'O', 'U', 'Á', 'É', 'Í', 'Ó', 'Ú'];

const palabra = palabras[Math.floor(Math.random() * palabras.length)];
const encontradas = palabra.toUpperCase().split('').filter(letra => vocales.includes(letra));

console.log(`Palabra: ${palabra}`);
console.log(`Vocales encontradas: ${encontradas.join(', ')}`);
console.log(`Cantidad de vocales: ${encontradas.length}`);

File: ejercicios-py/ejercicio3.py
import random
import re

palabras = [
    "Aceituna", "Murciélago", "Educación", "Aeropuerto", "Otorrinolaringólogo",
    "Euforia", "Aceite", "Paleontólogo", "Arquitectura", "Hipopótamo"
]

palabra = random.choice(palabras)
vocales = re.findall(r"[aeiouáéíóú]", palabra, re.IGNORECASE)

print(f"Palabra seleccionada: {palabra}")
print(f"Vocales encontradas: {', '.join(vocales)}")
print(f"Total de vocales: {len(vocales)}")